.github/actions/release/bump-versions/action.yml

```
@@ -0,0 +1,90 @@
+name: 'bump-versions'
+description: 'Bump versions in files for release target'
+inputs:
+  release-target:
+    description: "target of the release (rust|wasm)"
+    required: true
+  version:
+    description: "release name"
+    required: true
+
+
+runs:
+  using: "composite"
+  steps:
+    - uses: actions-rs/toolchain@v1
+      with:
+        toolchain: stable
+        profile: minimal
+        override: true
+
+    - name: Install cargo-release
+      uses: actions-rs/cargo@v1
+      with:
+        command: install
+        args: cargo-release
```

We _could_ use [cargo-workspaces](https://github.com/pksunkara/cargo-workspaces) to bump the versions instead since it allows us to disable the automatic commit.

```suggestion
    - name: Install cargo-workspaces
      uses: actions-rs/cargo@v1
      with:
        command: install
        args: --version ^0.2 cargo-workspaces
```

The version bump command would then be something like:

```
cargo workspaces version --no-git-commit --exact custom ${{ inputs.version }} -y
```

and we can remove the `release.toml` file.